# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1348079550 14400
#      Wed Sep 19 14:32:30 2012 -0400
# Node ID 2afbe42956829eb7b5990402ebee0e5ae98a99c6
# Parent  60090630cb765f565e8195a5eda1ff5753b9e343
correct name of include guard in generated builtin-defun-decls.h file

* mkbuiltins: Use octave_builtin_defun_decls_h for multiple inclusion
guard instead of octave_builtins_h.

diff --git a/libinterp/mkbuiltins b/libinterp/mkbuiltins
--- a/libinterp/mkbuiltins
+++ b/libinterp/mkbuiltins
@@ -42,18 +42,18 @@ case "$1" in
   ;;
 esac
 
 if $make_header; then
 
   cat << \EOF
 // DO NOT EDIT!  Generated automatically by mkbuiltins.
 
-#if !defined (octave_builtins_h)
-#define octave_builtins_h 1
+#if !defined (octave_builtin_defun_decls_h)
+#define octave_builtin_defun_decls_h 1
 
 #define DEFUN_DECL(name) \
   extern OCTINTERP_API octave_value_list \
   name (const octave_value_list& = octave_value_list (), int = 0);
 
 class octave_value_list;
 
 EOF
